instagram-app/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PostContainer from './PostContainer'
+import SearchBar from '../SearchBar/SearchBar'
+
+import dummyData from '../../dummy-data';
+
+
+
+
+class PostsPage extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      dummy: [],
+      searchPosts: []
+    };
+
+  }
+  componentDidMount(){
+    this.setState({dummy:dummyData});
+    this.setState({searchPosts:dummyData});
+  }
+searchHandler = event =>{
+  let Sdummy = this.state.dummy.filter(sUser =>{
+    if (sUser.username.includes(event.target.value)){
+      return sUser;
+    }
```

Comment:
You might want to rename `Sdummy` to something more declarative. Since it is meant to contain posts filtered by the provided search terms, perhaps a name like `filteredPosts` or `searchedPosts` would work more clearly here.